habitat/sims/habitat_simulator.py

```diff
@@ -451,3 +454,18 @@ def distance_to_closest_obstacle(self, position, max_search_radius=2.0):
         return self._sim.pathfinder.distance_to_closest_obstacle(
             position, max_search_radius
         )
+
+    @property
+    def up_vector(self):
```

Comment:
Let's add this method to Simulator API `core/simulator.py`.

Reply:
I see what you mean now. shortest_path_follower expects a HabitatSim object rather than an arbitrary Simulator object, so I don't know if this makes sense to add to the Simulator API. It is possible that different simulators will need different algorithms for shortest paths.

Reply:
@danielgordon10, yes we are overfitting to Habitat Sim here, but we will revisit the design choices when new simulators will need to be integrated.